thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/ThirdEyeAnomalyApplication.java

```diff
@@ -41,8 +41,8 @@
 import org.apache.pinot.thirdeye.datasource.DAORegistry;
 import org.apache.pinot.thirdeye.datasource.ThirdEyeCacheRegistry;
 import org.apache.pinot.thirdeye.datasource.pinot.resources.PinotDataSourceResource;
-import org.apache.pinot.thirdeye.detection.DetectionPipelineScheduler;
-import org.apache.pinot.thirdeye.detection.alert.DetectionAlertScheduler;
+import org.apache.pinot.thirdeye.scheduler.DetectionScheduler;
+import org.apache.pinot.thirdeye.scheduler.SubscriptionScheduler;
```

Comment:
I think these name changes are great for adding clarity and consistency

Reply:
@xiaohui-sun, yes, I have been consistently trying to enforce the usage of subscription rather than detection_alert in our code (this pr and in the past). detection_alert is a very confusing term.

The database still holds the references as detection_alert which is a little harder to migrate but eventually we should.